templates/publisher/market.html

```diff
@@ -40,15 +40,23 @@ <h1 class="u-no-margin--top">{{ title }}</h1>
         </div>
 
         {% with messages = get_flashed_messages(with_categories=true) %}
-          {% if messages %}
+          {% if messages or error_list %}
             <div class="row">
               <div class="col-12">
                 {% for category, message in messages %}
                   <div id="market-form-status" class="p-notification--{{ category }}" style="display: block">
                     <p class="p-notification__response">
                         {{ message }}
                     </p>
-                    </div>
+                  </div>
+                {% endfor %}
+
+                {% for error in error_list %}
+                  <div class="p-notification--negative" style="display: block">
```

Comment:
@bartaz FYI the `display: block` can be removed when you update Vanilla to `v1.6.5`

Reply:
@anthonydillon Great, thanks.